packages/taiko-client/pkg/rpc/beaconclient.go

```diff
@@ -95,13 +95,17 @@ func (c *BeaconClient) GetBlobs(ctx context.Context, time uint64) ([]*blob.Sidec
 		return nil, err
 	}
 
-	var sidecars *blob.SidecarsResponse
 	resBytes, err := c.Get(ctxWithTimeout, c.BaseURL().Path+fmt.Sprintf(sidecarsRequestURL, slot))
 	if err != nil {
 		return nil, err
 	}
 
-	return sidecars.Data, json.Unmarshal(resBytes, &sidecars)
```

Comment:
If `json.Unmarshal(resBytes, &sidecars)` failed, then `sidecars` is nil so `sidecars.Data` will cause panic.